packages/firestore/test/integration/api/get_options.test.ts

```diff
@@ -525,20 +566,15 @@ apiDescribe('GetOptions', persistence => {
 
   it('get non existing doc while offline with source=server', () => {
     return withTestDocAndInitialData(persistence, null, docRef => {
-      return (
-        docRef.firestore
-          .disableNetwork()
-          // attempt to get doc. Currently, this is expected to fail. In the
-          // future, we might consider adding support for negative cache hits so
-          // that we know certain documents *don't* exist.
-          .then(() => docRef.get({ source: 'server' }))
-          .then(
-            doc => {
-              expect.fail();
-            },
-            expected => {}
-          )
-      );
+      return docRef.firestore
+        .disableNetwork()
+        .then(() => docRef.get({ source: 'server' }))
+        .then(
+          doc => {
+            expect.fail();
+          },
+          expected => {}
+        );
```

Comment:
I know this is existing code, but it took me a while to parse this. The line breaks make it hard to understand that this is one function call with two arguments, and I believe they might not even be needed:

```
.then(() => expect.fail(), () => {});
```

This is extremely optional.

Reply:
Yeah...  this pattern exists throughout the file so I'm going to leave it as-is.